src/pages/iou/IOUDetailsModal.js

```diff
@@ -10,7 +10,6 @@ import {withNetwork} from '../../components/OnyxProvider';
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import Navigation from '../../libs/Navigation/Navigation';
 import ScreenWrapper from '../../components/ScreenWrapper';
-import * as IOU from '../../libs/actions/IOU';
 import * as Report from '../../libs/actions/Report';
```

Comment:
Is this file even accessed anywhere anymore? I dont think so

Reply:
Don't think so, but it's still in the Navigator stack. I'll leave it here for now and logged an issue for it - https://github.com/Expensify/App/issues/19167